MiaServiceDotNetLibrary/MiaServiceDotNetLibrary.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <TargetFramework>netcoreapp3.1</TargetFramework>
-        <PackageId>MiaServiceDotNetLibrary</PackageId>
+        <PackageId>PippoServiceDotNetLibrary</PackageId>
```

Comment:
```suggestion
        <PackageId>MiaServiceDotNetLibrary</PackageId>
```

Reply:
My bad! Now is set to the correct PackageId!